src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -58,6 +58,7 @@ function parseShadowsocksSessionConfigJson(maybeJsonText: string): ShadowsocksSe
     host: server,
     port: server_port,
     prefix,
+    error
```

Comment:
Error is not part of the session config. Please remove.

Error and the config are mutually exclusive. You communicate the error by throwing it.